tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "moduleResolution": "node",
+    "lib": [
+      "es6"
+    ],
+    "strict": true,
+    "noImplicitAny": false,
```

Comment:
I'd change it to `true` if we can.

Reply:
Switching this one causes a bunch of failures :/